federation/cmd/federation-controller-manager/app/controllermanager.go

```diff
@@ -77,7 +77,7 @@ func Run(s *options.CMServer) error {
 		glog.Errorf("unable to register configz: %s", err)
 	}
 	// Create the config to talk to federation-apiserver.
-	kubeconfigGetter := clustercontroller.KubeconfigGetterForSecret(FederationAPIServerSecretName)
+	kubeconfigGetter := util.KubeconfigGetterForSecret(FederationAPIServerSecretName)
```

Comment:
@nikhiljindal is this a prerequisite for starting controller manager? do we need always create secret for kubeconfig of federation controller plan? I keep this as is but want to let you know we can simply run ./federation-controller-manager --v=5 to start controller manager without any preconfiguration, but now I have to create secret for kubeconfig.